From: Michael Tokarev <mjt@tls.msk.ru>
Subject: drop "replace" dependency from libldb
Date: Wed, 27 Nov 2024 21:51:50 +0300
Forwarded: not-needed

When building as a sub-library within samba, where
libreplace is a private library, libldb.so will
have rpath pointing to the private samba dir.  Since
ldb actually does not use anything from libreplace,
just remove the dependency.

diff --git a/lib/ldb/wscript b/lib/ldb/wscript
index ab33f7784a6..41675c7ebe9 100644
--- a/lib/ldb/wscript
+++ b/lib/ldb/wscript
@@ -139,3 +139,3 @@ def build(bld):
                       COMMON_SRC + ' ' + LDB_MAP_SRC,
-                      deps='tevent LIBLDB_MAIN replace',
+                      deps='tevent LIBLDB_MAIN', # replace',
                       includes='include',
